SampleApp/DemoMapViewController.swift

```diff
@@ -86,31 +88,26 @@ class DemoMapViewController:  SampleMapViewController, MapMarkerSelectDelegate {
   }
 
   @objc private func changeMapLanguage() {
+    let languageIdByActionSheetTitle = [
+      "English": "en_US",
+      "French": "fr_FR",
+      "Japanese": "ja_JP",
+      "Hindi": "hi_IN",
+      "Spanish": "es_ES",
+      "Korean": "ko_KR",
+      "Italian": "it_IT",
+      ]
     let actionSheet = UIAlertController.init(title: "Map Language", message: "Choose a language", preferredStyle: .actionSheet)
-    actionSheet.addAction(UIAlertAction.init(title: "English", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "en_US"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "French", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "fr_FR"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "Japanese", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "ja_JP"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "Hindi", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "hi-IN"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "Spanish", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "es_ES"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "Korean", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "ko_KR"))
-    }))
-    actionSheet.addAction(UIAlertAction.init(title: "Italian", style: .default, handler: { [unowned self] (action) in
-      self.updateLocale(Locale.init(identifier: "it_IT"))
-    }))
+    for (actionTitle, languageIdentifier) in languageIdByActionSheetTitle {
+      actionSheet.addAction(UIAlertAction.init(title: actionTitle, style: .default, handler: { [unowned self] (action) in
```

Comment:
Kind of obnoxious you have to do it this way and that `UIAlertController` won't accept an array of actions. Would be a good use for `map()` function if that were the case. Ah well.